src/server.ts

```diff
@@ -30,6 +33,17 @@ passport.use(new BasicStrategy(
   }
 ));
 
+app.use(responseTime(function (req: Request, res: Response, time) {
+  const stat = (req.method + '/' + req.url.split('/')[1]).toLowerCase()
```

Comment:
Hmmm...  so this would take only the last thing in the url after the last / ?  Won't this miss out on what's most important (the actual endpoint names?).  Please tell me if I'm missing something here...